code/modules/client/preferences_savefile.dm

```diff
@@ -553,6 +553,15 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	else //We have no old flavortext, default to new
 		S["feature_flavor_text"]		>> features["flavor_text"]
 
+	if((S["silicon_flavor_text"] != "") && (S["silicon_flavor_text"] != null) && S["silicon_flavor_text"]) // dont change what isnt broken
```

Comment:
just do S["silicon_flavor_text"] >> features["silicon_flavor_text"]

Reply:
the whole conversion thing isn't needed since this is a new thing not an old thing that was converted